shared/src/arguments.rs

```diff
@@ -20,14 +20,13 @@ pub struct Arguments {
     #[structopt(long, env = "NODE_URL", default_value = "http://localhost:8545")]
     pub node_url: Url,
 
-    /// Timeout for web3 operations on the node in seconds.
+    /// Timeout for all http requests.
     #[structopt(
             long,
-            env = "NODE_TIMEOUT",
             default_value = "10",
             parse(try_from_str = duration_from_seconds),
         )]
-    pub node_timeout: Duration,
+    pub http_timeout: Duration,
```

Comment:
Maybe we should flag in the PR title that this requires a config change?

Reply:
I don't think we set it but anyway yes I should flag it.